topics/iterators/easy/array-iterator.tex

```diff
@@ -3,37 +3,39 @@
 
 \begin{lstlisting}
 Integer[] arr = {1, 2, 3, 4, 5, 6};
-Iterator<Integer> iter = new ArrayIterator<>(arr);
+Iterator iter = new ArrayIterator(arr);
```

Comment:
I think we didn't realize this earlier, but this actually won't work, unless I'm missing something. Java requires a type to be passed in. Let's discuss on Slack.